share/config.fish

```diff
@@ -254,8 +200,22 @@ end
 # Some things should only be done for login terminals
 # This used to be in etc/config.fish - keep it here to keep the semantics
 #
-
 if status --is-login
+    # OS X-ism: Call path_helper and set its output to $PATH and $MANPATH.
+    if test -x /usr/libexec/path_helper
```

Comment:
Actually, these days if you're checking an absolute path or simply want to confirm that an external command by a given name exists we prefer `command -sq /usr/libexec/path_helper` or `command -sq path_helper`. Although the latter won't work in this specific instance since */usr/libexec* is unlikely to be in `$PATH`. Also, neither `command -sq` or `type -q` lets you test and run the command.

Normally you could take advantage of the fact that `set` has special behavior: it returns the status of the subcommand as its status. So you might be inclined to do this:

```
if set -l lines (/usr/libexec/path_helper -c ^/dev/null)
    # whatever
end
```

Unfortunately at the present time you can't suppress the error output.